src/pages/home/sidebar/AllSettingsScreen.tsx

```diff
@@ -9,12 +11,22 @@ import useThemeStyles from '@hooks/useThemeStyles';
 import useWaitForNavigation from '@hooks/useWaitForNavigation';
 import useWindowDimensions from '@hooks/useWindowDimensions';
 import Navigation from '@libs/Navigation/Navigation';
+import {hasGlobalWorkspaceSettingsRBR} from '@libs/WorkspacesUtils';
 import * as Link from '@userActions/Link';
 import CONST from '@src/CONST';
 import type {TranslationPaths} from '@src/languages/types';
+import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import type {Policy, PolicyMembers} from '@src/types/onyx';
 
-function AllSettingsScreen() {
+type AllSettingsScreenOnyxProps = {
+    policies: OnyxCollection<Policy>;
+    policyMembers: OnyxCollection<PolicyMembers>;
+};
+
+type AllSettingsScreenProps = AllSettingsScreenOnyxProps;
```

Comment:
Do we need to create two types? 
```suggestion
type AllSettingsScreenProps = {
    policies: OnyxCollection<Policy>;
    policyMembers: OnyxCollection<PolicyMembers>;
};
```

Reply:
I've implemented it this way because withOnyx requires two types. It's implemented similarly in the`IFrame` component